test/test_bandedlu.jl

```diff
@@ -24,11 +24,7 @@ let
     @test BandedMatrices._promote_to_blas_type(typ, Int16)      == typ
     @test BandedMatrices._promote_to_blas_type(typ, Int8)       == typ
 
-    if VERSION < v"0.5.0-dev+5122"
```

Comment:
OK, in that case its probably better to leave the special case there until the bug is fixed.  (I'm guessing this is the issue involving KroneckerOperator?  So whatever it takes to help that along!)